backend/src/utils/adminUtils.ts

```diff
@@ -25,17 +25,17 @@ export const getAdminUserList = async (fastify: KubeFastifyInstance): Promise<st
   const adminGroups = getAdminGroups();
   const adminGroupsList = adminGroups
     .split(',')
-    .filter((groupName) => !groupName.startsWith('system:')); // do not include k8s defaults
-
+    .filter((groupName) => !groupName.startsWith('system:')) // do not include k8s defaults
+    .filter((groupName) => groupName != ''); // To prevent edge cases where groups are unset
   return getGroupUserList(fastify, adminGroupsList);
 };
 
 export const getAllowedUserList = async (fastify: KubeFastifyInstance): Promise<string[]> => {
   const allowedGroups = getAllowedGroups();
   const allowedGroupList = allowedGroups
     .split(',')
-    .filter((groupName) => !groupName.startsWith('system:')); // do not include k8s defaults
-
+    .filter((groupName) => !groupName.startsWith('system:')) // do not include k8s defaults
+    .filter((groupName) => groupName != ''); // To prevent edge cases where groups are unset
```

Comment:
Can you combine these two filters together?
`.filter((groupName) => groupName && !groupName.startsWith('system:'))`